types/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/regen-network/regen-ledger/types
+
```

Comment:
How can I fix this error?
```
go mod tidy
go: github.com/regen-network/regen-ledger/app@v0.0.0-00010101000000-000000000000 requires
        github.com/regen-network/regen-ledger/x/data@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

Reply:
Is this still relevant? If not, please resolve the conversation